opentelemetry-sdk/src/growable_array.rs

```diff
@@ -0,0 +1,374 @@
+/// The default max capacity for the stack portion of `GrowableArray`.
+const DEFAULT_MAX_INLINE_CAPACITY: usize = 10;
+/// The default initial capacity for the vector portion of `GrowableArray`.
+const DEFAULT_INITIAL_OVERFLOW_CAPACITY: usize = 5;
+
+#[derive(Debug, Clone, PartialEq)]
+/// A hybrid vector that starts with a fixed-size array and grows dynamically with a vector.
+///
+/// `GrowableArray` uses an internal fixed-size array (`initial`) for storing elements until it reaches
```

Comment:
```suggestion
/// `GrowableArray` uses an internal fixed-size array (`inline`) for storing elements until it reaches
```

Reply:
not a strong opinion, but `inline` in Rust mostly refers to the compiler optimization so maybe we can use something else to avoid confusion?

Reply:
I don't see the point of confusion.. Inline can refer to code being in-lined, or memory being inline (when using structs etc.)... But this is purely internal, so not a concern.

(I think I suggested inline, overflow from .NET runtimes's similar naming. .NET also has the compiler inlining 
optimization!)